utils/run_in_ga.sh

```diff
@@ -54,5 +54,24 @@ echo ::group::Setting up daos_admin
 echo ::endgroup::
 
 echo ::group::Container copy test
-./utils/node_local_test.py --no-root --test cont_copy
+export LD_LIBRARY_PATH=/opt/daos/prereq/debug/spdk/lib/
+./utils/node_local_test.py --no-root --memcheck no --test cont_copy
 echo ::endgroup::
+
+echo ::group::Fi test
+./utils/node_local_test.py --no-root fi
```

Comment:
(lint) Use semicolon or linefeed before 'fi' (or quote to make it literal). [SC1010]